test/problems/hello/submissions/run_time_error/test-memsize.cc.disabled

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
```

Comment:
Can you revert to the original contents? And add a comment as to why this was disabled?

Maybe also add a comment to the others with the reason for disabling.